install/helm/agones/templates/extensions-deployment.yaml

```diff
@@ -114,6 +114,8 @@ spec:
               fieldPath: metadata.namespace
         - name: CONTAINER_NAME
           value: "agones-extensions"
+        - name: READINESS_SHUTDOWN_DURATION
+          value: {{ .Value.agones.extensions.periodSeconds * .Value.agones.extensions.failureThreshold * 2 | quote }}
```

Comment:
You need the `s` I had in the example. So 

```
{{ mumble }}s
```

to indicate seconds, since it's a duration now.